.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Wildfly Cekit OpenShift Modules - Bats
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+     branches: [ master ]
+env:
+  LANG: en_US.UTF-8
+  STANDALONE_CONFIG_URL: https://raw.githubusercontent.com/jboss-container-images/jboss-eap-modules/7.3.x-legacy/jboss-eap-config-openshift/CD17.0/added/standalone-openshift.xml
+jobs:
+  bats:
+    name: Bats Shell Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: n1hility/cancel-previous-runs@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Setup required system packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install bats libxml2-utils nmap
+          sudo ln -s /usr/bin/ncat /usr/bin/nc
```

Comment:
Ubuntu 18.0.4 LTS doesn't have *ncat* which is what we really want. ncat is available in nmap, so we install that and link it to nc for the tests.

Using *netcat* from the default repos just hangs on ubuntu.